hack/tools/conversion-verifier/main.go

```diff
@@ -197,21 +197,21 @@ func main() {
 		}
 
 		if !storageType.IsHub() {
-			result = multierror.Append(result,
+			errs = append(errs,
 				errors.Errorf("type %q in package %q marked as storage version but it's not convertible, missing Hub() method", name, storageType.pkg.PkgPath),
 			)
 		}
 		for _, decl := range storageType.otherDecls {
 			if !decl.IsConvertible() {
-				result = multierror.Append(result,
+				errs = append(errs,
 					errors.Errorf("type %q in package %q it's not convertible, missing ConvertFrom() and ConvertTo() methods", name, decl.pkg.PkgPath),
 				)
 			}
 		}
 	}
 
-	if result != nil {
-		klog.Exit(result.Error())
+	if errs != nil {
+		klog.Exit(kerrors.NewAggregate(errs).Error())
```

Comment:
There is a downgrade in UX here from:
```
F0814 11:16:57.464635   80425 main.go:214] 2 errors occurred:
        * type "cluster.x-k8s.io/MachineDeployment" in package "sigs.k8s.io/cluster-api/api/v1beta1" marked as storage version but it's not convertible, missing Hub() method
        * type "cluster.x-k8s.io/MachineSetList" in package "sigs.k8s.io/cluster-api/api/v1beta1" marked as storage version but it's not convertible, missing Hub() method

```
to:

```
[type "cluster.x-k8s.io/MachineDeploymentList" in package "sigs.k8s.io/cluster-api/api/v1beta1" marked as storage version but it's not convertible, missing Hub() method, type "cluster.x-k8s.io/MachineHealthCheck" in package "sigs.k8s.io/cluster-api/api/v1beta1" marked as storage version but it's not convertible, missing Hub() method, type "cluster.x-k8s.io/MachineDeployment" in package "sigs.k8s.io/cluster-api/api/v1beta1" marked as storage version but it's not convertible, missing Hub() method]
make: *** [Makefile:628: verify-conversions] Error 1
```

But I'm fine with that as this tool is pretty low impact and isn't really user facing. We could implement our own string function to compensate thoug if needed.

Reply:
Can we add a simple for loop with fmt.Println + os.Exit at the end?

Reply:
Also len(errs) > 0 seems more idiomatic vs != nil